### PR TITLE
Expand rows

### PR DESCRIPTION
**Pull request by nimam-bentley**
https://github.com/iTwin/synchronization-report-react/pull/64
+ semver resolution

### CODE REVIEW

**Comment by aruniverse** [comment]
@nimam-bentley you prob want to merge with `main` first
also, why is this a "pre-release" and not just a regular one?

File: src/components/ProblemsTable.scss [hunk 1]
```diff
@@ -11,3 +11,7 @@
   gap: var(--iui-size-xs);
   justify-content: space-between;
 }
+
+:where(.iui-root, [data-iui-theme]) .iui-table-body .iui-table-row.active .iui-table-cell {
```
Comment (mayank99) [inline src/components/ProblemsTable.scss]: Please avoid targeting internal iui- classes. You can apply your custom className to a `<DefaultCell>` passed to `cellRenderer`. Or use a custom class in `rowProps` then target the cell using `> *`.

File: src/components/ProblemsTable.tsx [hunk 2]
```diff
@@ -156,6 +187,21 @@ export const ProblemsTable = ({
           Filter: tableFilters.TextFilter(),
           minWidth: 50,
           maxWidth: 250,
+          Cell: (row: CellProps<Report>) => {
+            return (
+              <div
+                className='iui-anchor'
```
Comment (mayank99) [inline src/components/ProblemsTable.tsx]: Please avoid using internal iui- classes. You can use the `<Anchor>` react component.

**Comment by mayank99** [comment]
Looks like #64 was reverted in https://github.com/iTwin/synchronization-report-react/commit/e246a9cf388f9b1442035a42367491a7fd6bdede and then unreverted (through force push?), so all the changes in _this_ PR are gone.

I still have the same comments - avoid targeting internal `iui-` classes. They are an implementation detail of iTwinUI-react and will most certainly change in the future, creating unnecessary churn for you.

File: src/components/ProblemsTable.scss [hunk 3]
```diff
@@ -12,6 +12,6 @@
   justify-content: space-between;
 }
 
-:where(.iui-root, [data-iui-theme]) .iui-table-body .iui-table-row.active .iui-table-cell {
-  background-color: var(--iui-color-background-accent-muted);
+.active-row > div {
+  background-color: var(--iui-color-background-accent-muted) !important;
```
Comment (Guillar1) [inline src/components/ProblemsTable.scss:16]: This classname needs a prefix and is the `!important` necessary?

**Review by Aidas-Vaiksnoras — APPROVED** [review]

**Pull request merged**